### DISCUSSION TITLE
Source code for examples from https://ui.shadcn.com/examples/dashboard

### DISCUSSION THREAD

**Discussion by reilly1993** [shadcn-ui/ui#3500] [General]
Is there any way to get the source code from the examples shown on the page to see how the shadcn/ui-team would put these UI's together as inspiration?

**Comment by MrBisquit** [comment]
https://github.com/shadcn-ui/ui/blob/main/apps/www/app/(app)/examples/dashboard/page.tsx

**Comment by thalesfsp** [comment]
Still, ,how to run the examples? For example, the email one?
